spring-cloud-dataflow-server-core/pom.xml

```diff
@@ -96,6 +101,12 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-jdbc</artifactId>
+			<exclusions>
```

Comment:
Have we run into any issue without this exclusion?

Reply:
Not really, but having two connection pools in the classpath isn't a good idea.